### DISCUSSION TITLE
itwin API GET project returns 404 Not Found for a project that exists

### DISCUSSION THREAD

**Discussion by PeterJBassett** [iTwin/itwinjs-core#2927] [Q&A] [answered]
Hello - I am working on developiong the FutureOn-Bentley connector (imodel-bridge). I am trying to synch a to a project and iModel set up in Plantsight. 

The Project and iModel exist (.https://qa-connect-imodelhubwebsite.bentley.com/Context/d19e5558-5a29-46b8-bb5e-67131f492a3c/iModel/d56a1253-dbc4-4b74-b9dd-1ebc11e42a04 ). 

When I try to read the project details using the itwin API ( https://developer.bentley.com/apis/projects/operations/get-project/ ). I get the following error:

GET https://api.bentley.com/projects/d19e5558-5a29-46b8-bb5e-67131f492a3c HTTP/1.1

    404 Not Found
    "error": {
        "code": "ProjectNotFound",
        "message": "Requested project not available."
    }

Why doesn't this work?

**Comment by aruniverse** [comment, marked as answer]
Your project & imodel live in qa, but you're making a call to the prod api. Do you have a project/imodel you can use in prod?

**Comment by PeterJBassett** [comment]
Thanks you very much for your help. We will set up a new project/imodel in prod.
